alts/src/main/java/io/grpc/alts/ComputeEngineChannelBuilder.java

```diff
@@ -84,16 +53,6 @@ protected NettyChannelBuilder delegate() {
 
   @VisibleForTesting
   ProtocolNegotiator getProtocolNegotiatorForTest() {
```

Comment:
This can be made  `static`? I understand this was old code but we can make it static without breaking old code.

Reply:
I had started to do so originally, but it's really neither here nor there as this method should really just be deleted. If I change it to static the linters will break the build, as you shouldn't call static methods on an instance variable. At that point the tests should be swapped over to using the credentials directly instead of the builder. Are you wanting me to do that now?

Reply:
(It looks like it would be easy to update the tests; just be a bit more to review.)

Reply:
I looked up and there is only one call to this method from a test (unless I missed others)

Reply:
I just deleted the method and the callers, here and in GoogleDefaultChannelBuilder and the credentials. They obviously served no purpose when static.